### ISSUE TITLE
tuna worker is offline 

### ISSUE THREAD

**Issue by zxkane** [tuna/opentuna#93]
run below command in manager,
```bash
tunasynctl workers -p 80
[
  {
    "id": "tunasync-worker",
    "url": "",
    "token": "",
    "last_online": "2020-09-01T02:52:27.740256625Z"
  }
]
```

But both worker and manager are running well.

### Environment

  - **CDK CLI Version:** 
  - **Framework Version:**
  - **Node.js Version:** 
  - **OS               :**

### Other

--- 

This is :bug: Bug Report

**Comment by jiegec** [comment]
It seems that tunasync manager starts at 20-09-07 09:29:47, but worker starts at 20-09-07 09:26:29, so worker failed to register with the manager.

**Comment by zxkane** [comment]
The status of jobs still can be synced with manager. The manager should
update the info of worker as well.

Chen <notifications@github.com> 于 2020年9月10日周四 下午7:45写道：

**Comment by jiegec** [comment]
I will submit a pr to tuna/tunasync to fix this.

**Comment by jiegec** [comment]
Closed as of #95
